README.md

```diff
@@ -176,7 +176,7 @@ To use an external Prometheus instance for a certain project, a secret containin
 
 ```console
 PROMETHEUS_USER=test
- PROMETHEUS_PASSWORD=test-password
+PROMETHEUS_PASSWORD=test-password
```

Comment:
the space is in here for purpose, as anything prefixed with a space is not stored in your bash history. For a password, it's crucial that it is not stored in the bash history.

Reply:
Got it, I will close this PR.